### DISCUSSION TITLE
需求请求：读取ptr记录，在连接界面添加客户端名称列；活动下方IP地址变更为客户端名称

### DISCUSSION THREAD

**Discussion by yyysuo** [vernesong/OpenClash#3337] [Ideas]
可否读取主路由本身的ptr记录，把内网ip地址显示为客户端名称，同时添加客户端名称列，这样比较方便看系统里面哪些设备在跑流量，对于没有ptr记录的设备，可否在系统指定目录自己维护自定义显示的设备名称，如果有ptr记录的话，优选显示自定义的设备名称。非常感谢。

图就借用meta版本的图了。

![image](https://github.com/vernesong/OpenClash/assets/10081071/9edd41a4-40fa-4363-9ae8-20d13319bf68)

**Comment by ghost** [comment]
这个应该是内核和面板的事吧。应该去那边提。
而且这个基本是针对路由器上跑clash的功能，只能看开发者心情了。
